roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -38,6 +38,11 @@
   when: create_ha_unit_files | changed
 # end workaround for missing systemd unit files
 
+- name: Preserve Master API Proxy Config options
+  command: grep PROXY /etc/sysconfig/{{ openshift.common.service_type }}-master-api
+  register: master_api_proxy
+  when: openshift.master.ha is defined and openshift.master.ha | bool and openshift_master_cluster_method == "native"
```

Comment:
You mean on disk value for openshift.common.http_proxy and friends or on disk value by parsing /etc/sysconfig/docker and /etc/sysconfig/atomic-openshift-master* ? Those two will actually be slightly different as the docker config doesn't know how to handle CIDR addresses and the master config does.